RecoMuon/TrackerSeedGenerator/python/customizeOIseeding.py

```diff
@@ -0,0 +1,33 @@
+import FWCore.ParameterSet.Config as cms
+
+from HLTrigger.Configuration.common import *
+
+def customizeOIseeding(process):
+    """
+    - adds doublet-like HB seeds (maxHitDoubletSeeds)
+    - HL seeds from two trajectories (IP, MuS) are considered separate types
+    - Number of seeds of each type and error SF for HL seeds  can be determined individually for each L2 muon using a DNN
+    """
+    process.hltIterL3OISeedsFromL2Muons = cms.EDProducer( "TSGForOIDNN",
+        MeasurementTrackerEvent = cms.InputTag("hltSiStripClusters"),
```

Comment:
```suggestion
    import RecoMuon.TrackerSeedGenerator.tsgForOIDNN_cfi as _mod
    process.hltIterL3OISeedsFromL2Muons = _mod.tsgForOIDNN.clone(
        MeasurementTrackerEvent = "hltSiStripClusters",
```
- reuse the auto-generated config made by the `fillDescriptions`
-  please drop type specifications for all parameters which already exist. This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.

Reply:
please check 
https://github.com/cms-sw/cmssw/pull/35237/files#r712601310
This part is apparently still not addressed

Reply:
Done!